mercury-operator/src/main/resources/v1/customer-offer/customer-offer-v1-direct.yaml

```diff
@@ -0,0 +1,35 @@
+# camel-k: language=yaml
+# camel-k: dependency=camel-quarkus-protobuf
+# camel-k: dependency=mvn:com.redhat.mercury:customer-offer-common:1.0.2-SNAPSHOT
```

Comment:
maybe we can use the maven resources plugin to not have the version hardcoded here.
https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html